scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -473,6 +478,13 @@
                       {:max-synthetic-pods max-pods-outstanding
                        :synthetic-pods num-synthetic-pods}))
 
+          (set-counter-fn "total-pods" total-pods)
+          (set-counter-fn "max-total-pods" max-total-pods)
+          (set-counter-fn "total-nodes" total-nodes)
+          (set-counter-fn "max-total-nodes" max-total-nodes)
+          (set-counter-fn "num-synthetic-pods" num-synthetic-pods)
+          (set-counter-fn "max-pods-outstanding" max-pods-outstanding)
```

Comment:
For consistency with the other metric names, maybe `total-synthetic-pods` and `max-total-synthetic-pods`?

Reply:
thanks, updated